api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -33,6 +26,7 @@
  * @see TracerProvider
  * @see ContextPropagators
  */
+@SuppressWarnings("deprecation") // Remove after deleting OpenTelemetry SPI
```

Comment:
Wait up. We're deleting the global altogether? I think I might have missed a memo. I'm assuming this is a mistake?

Reply:
This is a suppresswarnings, no a Deprecated :P

Reply:
oh. yes. :disappear: